src/Administration/Resources/app/administration/src/module/sw-order/component/sw-order-general-info/index.js

```diff
@@ -189,17 +188,7 @@ export default {
 
     methods: {
         createdComponent() {
-            const tags = cloneDeep(this.order.tags);
-
-            this.tagCollection = new EntityCollection(
-                this.order.tags.source,
-                this.order.tags.entity,
-                Shopware.Context.api,
-                null,
-                tags,
-                tags.length,
-            );
-
+            this.tagCollection = this.order.tags;
```

Comment:
This changes the behavior how the data get handled.

Previously:
- cloned the value deeply
- created a new EntityCollection based on the cloned values

Now: 
- References the tags directly in the order

I can imagine that the previous developer did this on purpose. Changing this could lead to several unknown side effects

Reply:
I reverted this, as the developer who changed this, was trying to fix the exact same issue, but apparently it did not work, so I thought it could be reverted. In the customers module, the same component also just gets passed `customer.tags` in the same way. But I can change it again. 🤷‍♂️

Reply:
Reverted the changes to the previous state.